src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -1,62 +1,24 @@
-import React, { useEffect, useMemo, useState } from 'react'
+import React from 'react'
 import { useParams } from 'react-router'
 
-import { getOrder, RawOrder } from 'api/operator'
+import { useOrderAndErc20s } from 'hooks/useOperatorOrder'
+
+import { ORDER_QUERY_INTERVAL } from 'apps/explorer/const'
 
 import { OrderWidgetView } from './view'
-import { useMultipleErc20 } from 'hooks/useErc20'
-import { useNetworkId } from 'state/network'
-import { Network } from 'types'
 
 export const OrderWidget: React.FC = () => {
-  // TODO: move order loading to a hook
-  const networkId = useNetworkId()
-  const [order, setOrder] = useState<RawOrder | null>(null)
-  const [isLoading, setIsLoading] = useState(false)
-  const [error, setError] = useState('')
-  // TODO: load fills from a hook
-
   const { orderId } = useParams<{ orderId: string }>()
 
-  useEffect(() => {
-    // TODO: get network from a hook or something
-    setIsLoading(true)
-
-    if (!networkId) {
-      return
-    }
-
-    getOrder({ networkId, orderId })
-      .then((order) => {
-        console.log(`got order ${order} for network ${networkId}`)
-        setOrder(order)
-        setError('')
-      })
-      .catch((e) => setError(e.message))
-      .finally(() => setIsLoading(false))
-  }, [networkId, orderId])
-
-  // TODO: this is just for testing. The hooks will not be here
-  const networkIdOrDefault = networkId ?? Network.Mainnet
-
-  const { isLoading: isErc20sLoading, value: erc20s } = useMultipleErc20({
-    addresses: [order?.buyToken || '', order?.sellToken || ''].filter(Boolean),
-    networkId: networkIdOrDefault,
-  })
-  const [buyToken, sellToken] = useMemo(() => {
-    const buyToken = erc20s[order?.buyToken || ''] || undefined
-    const sellToken = erc20s[order?.sellToken || ''] || undefined
-
-    return [buyToken, sellToken]
-  }, [order?.buyToken, order?.sellToken, erc20s])
+  const { order, buyToken, sellToken, isLoading, error } = useOrderAndErc20s(orderId, ORDER_QUERY_INTERVAL)
```

Comment:
Related to my comment above. This hooks looks fairly easy to use. Although one could think that buyToken, sellToken are part of the order. So ideally, the use would be:
```
const { order, isLoading, error } = useOrderAndErc20s(orderId, ORDER_QUERY_INTERVAL)

order.sellTokenAddress // address, we can leave it or kill this prop since it's derived from the sellToken
order.sellToken // token
```